### ISSUE TITLE
Syntax issue for output in Structs examples

### ISSUE THREAD

**Issue by MrL1605** [elixirschool/elixirschool#2071]
There is issue in some of examples of Structs where the output starts with `#` instead of `%`.
Eg. in [Module.md Line 86](https://github.com/elixirschool/elixirschool/blame/master/en/lessons/basics/modules.md#L86)
```elixir
#Example.User<name: "Sean", roles: [], ...>
```
should be
```elixir
%Example.User<name: "Sean", roles: [], ...>
``` 
Highlighter also shows error as shown below in the screenshot.

![Screenshot from 2019-10-20 18-35-13](https://user-images.githubusercontent.com/7734245/67160511-3786a080-f36f-11e9-8178-45dfd648d6bc.png)

Also, let me know If I can contribute.
I would like to fix this as part of Hacktober Fest.

**Comment by doomspork** [comment]
This is great @MrL1605! Please do contribute a fix, we'd love to have your help ❤️ 

If this change can be applied to translated lessons, let's do that as well 😁

**Comment by MrL1605** [comment]
Sure, I would do it for all languages.

Also, I did a grep and also found same issue in all mentions of [Ecto -> Changesets](https://github.com/elixirschool/elixirschool/blame/master/en/lessons/ecto/changesets.md#L21).
Should I fix that as well?

**Comment by doomspork** [comment]
That'd be awesome @MrL1605! In the interest of Hacktoberfest if you wanted to submit this as 2 different PRs I'd be happy to accept both:

1. Correcting the `basics/module` lesson across all languages
2. Correcting the `changeset` lesson across all languages

**Comment by MrL1605** [comment]
> if you wanted to submit this as 2 different PRs I'd be happy to accept both:

That would be too easy. I would like to earn the T-shirt the hard way. :smiling_imp:

**Comment by SophieDeBenedetto** [comment]
Hey @MrL1605! Thanks so much for your contributions 🚀 🚀 🚀 
Since this issue seems to be resolved by #2073, I'm going to close this issue.
